NOTICE.md

```diff
@@ -0,0 +1,4 @@
+When compiled with the following features, distributed with `GPL3` license
+- `tuic`
```

Comment:
do we actually have a feature flag for that?

Reply:
not yet（